### PR TITLE
Typo in the heading. "Dashboord" should be "Dashboard."

### PR DESCRIPTION
**Pull request by nitin611**
Issue: <h1>Web Dashboord</h1>
Solution: <h1>Web Dashboard</h1>

Incorrect attribute name "Licence" should be "License."

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   </a>
-  <a href="https://opensource.org/license/gpl-3-0/">
-    <img src="https://img.shields.io/badge/license-GPLv3-blue" alt="Licence" />
+  <a href="https://opensource.org/licenses/gpl-3.0/">
+    <img src="https://img.shields.io/badge/license-GPLv3-blue" alt="License" />
   </a>
 </div>
 
-![](https://i.imgur.com/waxVImv.png)
+<img src="https://i.imgur.com/waxVImv.png" alt="Web Dashboard" />
```
Comment (ahmfaiz) [inline README.md]: As this is a Markdown file, markdown code is preferred. Above HTML code was used for alignment reasons.
Kindly edit your pr.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
   <h1>The <a href="https://github.com/ahmfaiz?tab=repositories&q=wire&type=&language=&sort=">WIRE</a> Project</h1>
   <p>
     Web Info Retrieval Engine
   </p>
   <a href="https://opensource.org/">
     <img src="https://badges.frapsoft.com/os/v3/open-source.svg?v=103" alt="Open Source Love" />
   </a>
-  <a href="https://opensource.org/license/gpl-3-0/">
-    <img src="https://img.shields.io/badge/license-GPLv3-blue" alt="Licence" />
+  <a href="https://opensource.org/licenses/gpl-3.0/">
```
Comment (ahmfaiz) [inline README.md]: The link is correct and default one. Does not need changing.
Kindly edit your pr.

**Review by ahmfaiz — COMMENTED** [review]
Kindly edit pr and resubmit.

File: README.md (hunk 2 above)
Comment (nitin611) [reply README.md:11]: i have not changes the link there was error in this : - Incorrect attribute name "Licence" should be "License."

**Pull request merged**